src/main/java/org/prebid/server/bidder/BidderCatalog.java

```diff
@@ -22,11 +22,19 @@
     private final Map<String, BidderDeps> bidderDepsMap;
     private final Map<String, String> deprecatedNameToError = new HashMap<>();
     private final Map<String, String> aliases = new HashMap<>();
+    private final Set<String> activeCookieFamilyNames;
 
     public BidderCatalog(List<BidderDeps> bidderDeps) {
         bidderDepsMap = Objects.requireNonNull(bidderDeps).stream()
                 .collect(Collectors.toMap(BidderDeps::getName, Function.identity()));
 
+        activeCookieFamilyNames = bidderDeps.stream()
```

Comment:
i believe this `Set` should be initialized in `SetuidHandler` constructor.
Maybe it is redundant to expand `BidderCatalog` for one-time action.

Reply:
Will do